packages/cli/src/models/dependency/Dependency.ts

```diff
@@ -74,6 +74,15 @@ export default class Dependency {
     return this.fromNameWithVersion(nameAndVersion);
   }
 
+  public static splitNameAndVersion(nameAndVersion: string): [string, string] {
```

Comment:
```suggestion
    public static splitNameAndVersion(nameAndVersion: string): [string, string] {
    const isOrg = nameAndVersion.startsWith('@');
    const [name, version] = (isOrg ? nameAndVersion.slice(1) : nameAndVersion).split('@');
    return [isOrg ? `@${name}` : name, version];
  }
```

Reply:
Hmm why the extra indentation? Looks odd.

Reply:
A glitch during copy-paste. Github actually doesn't allow to make multiple line suggestions yet.